sdk/eventhub/Azure.Messaging.EventHubs.Experimental/src/Producer/IdempotentProducer.cs

```diff
@@ -206,10 +206,7 @@ protected IdempotentProducer() : base()
         /// </remarks>
         ///
         public virtual new async Task<PartitionPublishingProperties> GetPartitionPublishingPropertiesAsync(string partitionId,
-                                                                                                           CancellationToken cancellationToken = default)
-        {
-            var coreProperties = await base.GetPartitionPublishingPropertiesAsync(partitionId, cancellationToken).ConfigureAwait(false);
-            return new PartitionPublishingProperties(coreProperties.IsIdempotentPublishingEnabled, coreProperties.ProducerGroupId, coreProperties.OwnerLevel, coreProperties.LastPublishedSequenceNumber);
-        }
+                                                                                                           CancellationToken cancellationToken = default) =>
+            await base.GetPartitionPublishingPropertiesAsync(partitionId, cancellationToken).ConfigureAwait(false);
```

Comment:
nit: spacing looks off.

Reply:
Double-checked it in the [full view](https://github.com/Azure/azure-sdk-for-net/blob/68bd08e53c39d49ccc405e6a24e2dd1d0644bb39/sdk/eventhub/Azure.Messaging.EventHubs.Experimental/src/Producer/IdempotentProducer.cs) and spacing appears to line up.